Sources/TabBarCoordinator.swift

```diff
@@ -33,27 +33,12 @@ class TabBarCoordinator: NSObject {
 
     @objc func updateTheme() {
         editToolbar.backgroundColor = PresentationTheme.current.colors.tabBarColor
-        //Setting this in appearanceManager doesn't update tabbar and UINavigationbar of the settingsViewController on change hence we do it here
+        //Setting this in appearanceManager doesn't update tabbar on change hence we do it here
         tabBarController.tabBar.isTranslucent = false
         tabBarController.tabBar.backgroundColor = PresentationTheme.current.colors.tabBarColor
         tabBarController.tabBar.barTintColor = PresentationTheme.current.colors.tabBarColor
         tabBarController.tabBar.itemPositioning = .fill
-        tabBarController.viewControllers?.forEach {
```

Comment:
all this code was required only for updating `largeTitleTextAttributes` to avoid lag in `SettingsController`. All other controllers had lag.